metadata.rb

```diff
@@ -12,6 +12,6 @@
 
 recipe 'memcached::default', 'Installs and configures memcached'
 
-source_url 'https://github.com/chef-cookbooks/memcached'
-issues_url 'https://github.com/chef-cookbooks/memcached/issues'
-chef_version '>= 12.7'
+source_url 'https://github.com/sous-chefs/memcached'
+issues_url 'https://github.com/sous-chefs/memcached/issues'
+chef_version      '>= 13.8'
```

Comment:
What's the reasoning for 13.8?

Reply:
12.7 is ETL so it should probably be `'>= 13.0'` instead